client/src/main/java/com/tigrisdata/db/client/service/StandardTigrisDatabase.java

```diff
@@ -79,6 +83,41 @@ public CreateCollectionResponse createCollection(
     }
   }
 
+  @Override
+  public TigrisDBResponse createCollectionsInTransaction(List<URL> collectionsSchemas)
```

Comment:
do we need this and below as a separate method? let's use default as transaction?

Reply:
removed non transactional create collection method. I will update async interface in later PR.